desktop/linux/index.md

```diff
@@ -1,104 +1,172 @@
 ---
-description: Docker Desktop for Linux (Beta)
-keywords: docker, Desktop for linux, beta, tech preview
-title: Docker Desktop for Linux (Beta)
+description: Getting Started
+keywords: linux, tutorial, run, docker, local, machine
+redirect_from:
+- /docker-for-linux
+title: Docker Desktop for Linux user manual
 ---
 
-Welcome to the Docker Desktop for Linux (Beta). The Beta program is aimed at early adopters who would like to try Docker Desktop for Linux and provide feedback.
+Welcome to Docker Desktop! The Docker Desktop for Linux user manual provides information on how to configure and manage your Docker Desktop settings.
 
-Docker Desktop is an easy-to-install application that enables you to build and
```

Comment:
Will this be the main page people find when they google for Docker Desktop for Linux? If so, maybe we should keep some of the blurb ("Docker Desktop is an easy-to-install application that enables you to build and share containerized applications and microservices") so that people know what it is.